src/tests/golden/testdata/embeddedrules/precedence.php

```diff
@@ -54,4 +63,13 @@ function precedence_lhs_good($x, $mask) {
 
   $_ = (($x & $mask) != 0) | 0x02;
   $_ = (precedence_foo() !== 0) & 0x02;
+
+  $_ = ($mask > $z) ?? $x;
+  $_ = ($mask < $z) ?? $x;
+  $_ = ($mask <= $z) ?? $x;
+  $_ = ($mask >= $z) ?? $x;
+  $_ = ($mask == $z) ?? $x;
+  $_ = ($mask != $z) ?? $x;
+  $_ = ($mask === $z) ?? $x;
+  $_ = ($mask !== $z) ?? $x;
```

Comment:
Please add forms like `$x ?? $mask > $z` so we don't screw this up later as these forms are valid.